src/Faker/Provider/vi_VN/PhoneNumber.php

```diff
@@ -5,24 +5,29 @@
 class PhoneNumber extends \Faker\Provider\PhoneNumber
 {
     protected static $areaCodes = array(
-        76, 281, 64, 781, 240, 241,
-        75, 650, 56, 651, 62, 780,
-        26, 710, 511, 500, 510, 230,
-        61, 67, 59, 219, 351, 4,
-        39, 320, 31, 711, 218, 321,
-        8, 58, 77, 60, 231, 25,
-        20, 63, 72, 350, 38, 30,
-        68, 210, 57, 52, 510, 55,
-        33, 53, 79, 22, 66, 36,
-        280, 37, 54, 73, 74, 27,
-        70, 211, 29,
+        // Updated new area code on 17/06/2018
+        // http://vietnamnet.vn/vn/cong-nghe/vien-thong/danh-sach-ma-vung-dien-thoai-moi-cac-tinh-thanh-355763.html
```

Comment:
Does this source comply with the license of this package?

Reply:
This source of information is reliable by the state, it is a page that the number of vietnamese access large. And this source of information is shared and reused everywhere